File: 알고리즘/김연우/JAVA/10952.java
import java.util.Scanner;
 
public class Main {
	public static void main(String args[]){
		
		Scanner in=new Scanner(System.in);
				
		while(true){
		
			int A=in.nextInt();
			int B=in.nextInt();
		
			if(A==0 && B==0){
				in.close();
				break;
			}
			System.out.println(A+B);
		}
	}
}

File: 알고리즘(백준)/김연우/Kotlin/10952.kt
import java.util.Scanner

fun main(args: Array<String>){
    val sc: Scanner = Scanner(System.`in`)

    while (true){

        var a = sc.nextInt()
        var b = sc.nextInt()

        if (a == 0 && b == 0){
            break
        }
        println(a+b)

    }
}
